Sources/SwifterSwift/SwiftStdlib/RawRepresentableExtensions.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  RawRepresentableExtensions.swift
+//  SwifterSwift
+//
+//  Created by 方林威 on 2020/7/1.
+//  Copyright © 2020 SwifterSwift
+//
+
+// MARK: - Initializers
+public extension RawRepresentable {
+    
```

Comment:
This is giving you a SwiftLint error.
```suggestion

```

Reply:
Thank you for your suggestion, I will pay attention to the specification of SwiftLint next time